src/unix/linux_like/linux/mod.rs

```diff
@@ -1262,6 +1263,15 @@ pub const SCHED_RESET_ON_FORK: ::c_int = 0x40000000;
 
 // netinet/in.h
 // NOTE: These are in addition to the constants defined in src/unix/mod.rs
+
+/// Multipath TCP
+pub const IPPROTO_MPTCP: ::c_int = 262;
+#[deprecated(
+    since = "0.2.79",
+    note = "IPPROTO_MAX depends on the kernel version on the target system"
+)]
+pub const IPPROTO_MAX: ::c_int = 263;
```

Comment:
This change will affect the users who use 5.5 or lower kernel, right? If so, I'd avoid increasing it in this PR and just put a deprecation note like "this value will be changed following to the upstream change after a few releases.".

Reply:
Yes, in these cases the value defined here won't match the actual value used by the kernel anymore. However, given that users might use `IPPROTO_MAX` to validate `IPPROTO_*` values, it think it would be preferable if the value would reflect the definition of `IPPROTO_MPTCP`.

Reply:
So, anyway I think we should tell users this change before we go ahead. Generally, I wouldn't like to land any breaking changes without notice.

Reply:
I understand that the change is less than ideal. For the time being, I set it back to its original value. If you'd like I will create a new pull request which will increase it again and can be merged as soon as other breaking changes are introduced.